backend/app/controllers/concerns/session.rb

```diff
@@ -0,0 +1,10 @@
+module Session
+
+  def authenticate
+    head :unauthorized if current_user.blank? 
+  end
+
+  def current_user
+    User.find_by(uid:params[:uid])
```

Comment:
current_userメソッドは複数回アクセスされるのでメモ化しておかないとDBへの負荷が大変なことになります！
```suggestion
   @_current_user ||= User.find_by(uid: params[:uid])
```